samples/XCT.MauiCompat.Sample/Directory.Build.props

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This is a hack to avoid the project to import the SDK Build.props. This was causing a lot of issues